themes-default/slim/views/config_search.mako

```diff
@@ -17,88 +22,185 @@ const startVue = () => {
+                    verifyCert: ${js_bool(app.TORRENT_VERIFY_CERT)},
+                    paused: ${js_bool(app.TORRENT_PAUSED)},
+                    highBandwidth: ${js_bool(app.TORRENT_HIGH_BANDWIDTH)},
                 },
                 nzb: {
+                    enabled: ${js_bool(app.USE_NZBS)},
+                    dir: ${json.dumps(app.NZB_DIR)},
                     method: '${app.NZB_METHOD}',
                     nzbget: {
+                        useHttps: ${js_bool(app.NZBGET_USE_HTTPS)},
                         host: '${app.NZBGET_HOST}',
                         username: '${app.NZBGET_USERNAME}',
                         password: '${app.NZBGET_PASSWORD}',
-                        useHttps: '${app.NZBGET_USE_HTTPS}',
-                        testStatus: 'Click below to test'
+                        testStatus: 'Click below to test',
+                        category: '${app.NZBGET_CATEGORY}',
+                        categoryBacklog: '${app.NZBGET_CATEGORY_BACKLOG}',
+                        categoryAnime: '${app.NZBGET_CATEGORY_ANIME}',
+                        categoryAnimeBacklog: '${app.NZBGET_CATEGORY_ANIME_BACKLOG}',
+                        priority: ${app.NZBGET_PRIORITY},
+                        priorityOptions: {
+                            'Very low': -100,
+                            'Low': -50,
+                            'Normal': 0,
+                            'High': 50,
+                            'Very high': 100,
+                            'Force': 900
+                        }
                     },
                     sabnzbd: {
                         host: '${app.SAB_HOST}',
                         username: '${app.SAB_USERNAME}',
                         password: '${app.SAB_PASSWORD}',
                         apiKey: '${app.SAB_APIKEY}',
-                        testStatus: 'Click below to test'
+                        testStatus: 'Click below to test',
+                        category: '${app.SAB_CATEGORY}',
+                        categoryBacklog: '${app.SAB_CATEGORY_BACKLOG}',
+                        categoryAnime: '${app.SAB_CATEGORY_ANIME}',
+                        categoryAnimeBacklog: '${app.SAB_CATEGORY_ANIME_BACKLOG}',
+                        forced: ${js_bool(app.SAB_FORCED)}
                     }
                 },
                 httpAuthTypes: {
                     none: 'None',
                     basic: 'Basic',
                     digest: 'Digest'
                 },
-                searchIntervals: JSON.parse('${json.dumps(app.PROPERS_SEARCH_INTERVAL)}'),
+
+                // Episode Search: General Config
+                randomizeProviders: ${js_bool(app.RANDOMIZE_PROVIDERS)},
+                downloadPropers: ${js_bool(app.DOWNLOAD_PROPERS)},
+                checkPropersInterval: '${app.CHECK_PROPERS_INTERVAL}',
+                // propersSearchIntervals: JSON.parse('${json.dumps(app.PROPERS_SEARCH_INTERVAL)}'),
```

Comment:
Should this be removed or uncommented?

Reply:
Basically the Python/Mako version of it unnecessarily used it.
So - removed.
I thought about it, I wasn't sure before but now I am - we don't need these values, since the values & labels we need are all in `propersIntervalLabels`.

```py
# This is to get the actual time in minutes for each option.
PROPERS_SEARCH_INTERVAL = {'15m': 15, '45m': 45, '90m': 90, '4h': 4 * 60, 'daily': 24 * 60}
# This has both the values and labels we need.
PROPERS_INTERVAL_LABELS = {'daily': '24 hours',
                           '4h': '4 hours',
                           '90m': '90 mins',
                           '45m': '45 mins',
                           '15m': '15 mins'
                           }
```